x/feegrant/client/cli/cli_test.go

```diff
@@ -61,15 +62,14 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	}
 
 	fee := sdk.NewCoin("stake", sdk.NewInt(100))
-	duration := 365 * 24 * 60 * 60
 
 	args := append(
 		[]string{
 			granter.String(),
 			grantee.String(),
 			fmt.Sprintf("--%s=%s", cli.FlagSpendLimit, fee.String()),
 			fmt.Sprintf("--%s=%s", flags.FlagFrom, granter),
-			fmt.Sprintf("--%s=%v", cli.FlagExpiration, duration),
+			fmt.Sprintf("--%s=%s", cli.FlagExpiration, getFormattedExpiration(365*24*60*60)),
```

Comment:
let's create constants for this values (the numbers): 
+ `oneYear`, `oneDay` ...

Reply:
up